source/_patterns/01-molecules/navigation/view-selector~with-tabs.json

```diff
@@ -1,6 +1,4 @@
 {
-  "primaryUrl": "#primaryListing",
```

Comment:
There is a lot of refactoring in order to avoid presenting the view-selector with links if it is a tab selector. My view is that the links still make sense. I know we hide them for non-js but semantically it still works to have the href of the link to reference the fragment id of the section that is being shown on click. 

Preserving the links will avoid the refactor of the yaml, I presume. Let's discuss tomorrow.